nih-sparc/sparc-app
# Add icons for sharing news and events

# Description

The purpose of this PR is to add icons for sharing news and events.

## Type of change


# How Has This Been Tested?

- Go to the news and events page
- Click on one of the news items to go to its page
- At the bottom of the right sidebar, the share buttons should now be icons

# Checklist:
